test/create-test-server.js

```diff
@@ -106,3 +106,18 @@ test('support returning body directly', async t => {
 	t.deepEqual(bodyJson, { foo: 'bar' });
 	t.deepEqual(bodyAsync, 'bar');
 });
+
+test('support returning body directly without wrapping in function', async t => {
+	const server = await createTestServer();
+
+	server.get('/foo', 'bar');
+	server.get('/bar', ({ foo: 'bar' }));
```

Comment:
Out of interest, why do you wrap the object in parenthesis here?

Reply:
You can't return an object otherwise, else it becomes a regular function :).